Src/Particle/AMReX_ParticleContainerI.H

```diff
@@ -701,87 +722,142 @@ ParticleContainer<NStructReal, NStructInt, NArrayReal, NArrayInt, Allocator>
             virts_offset += np;
         }
     }
-
     if (aggregation_type == "Cell")
     {
+        //Components would be based on
+        MultiFab mf;
```

Comment:
You could do this all on the same line, instead of default constructing + calling define.

Reply:
Removed use of define